### PR TITLE
OCPQE-20101: Add DR test profile on OSP 16.2

### PR DESCRIPTION
**Pull request by wangke19**

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by gangwgr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-azure-ipi-private-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-private-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-openstack-ipi-f28-disasterrecovery

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -2149,6 +2149,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-upi
+- as: openstack-ipi-f28-disasterrecovery
+  cron: 11 1 11 * *
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      ROTATE_CLOUD_CREDENTIALS: "true"
+      TEST_IMPORTANCE: LEVEL0
```
[the remark this answers is not in the document]
Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2158]: Thank you for you informaton, will update jobs.

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-openstack-ipi-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openstack-ipi-f28-disasterrecovery

**Comment by wangke19** [comment]
Etcd and Apiserver DR cases have been executed with the cluster of OpenStack. Some of them ran into failures, that's another problem we need to solve.

**Comment by wangke19** [comment]
/assign @geliu2016 
Please help to review the pr.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -1936,6 +1936,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: openstack-ipi-f28-disasterrecovery
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1939]: 4.13 reached EOL. Most of the jobs there are running with `f60`. It's better to align the test with the others. Of course, feel free to customize if you have a specific test requirements.

Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: Updated with `f60`.

**Pull request merged**